src/code42cli/cmds/devices.py

```diff
@@ -0,0 +1,399 @@
+from collections import OrderedDict
+from datetime import date
+from datetime import datetime
+
+import click
+from pandas import DataFrame
+from pandas import to_datetime
+from pandas import to_timedelta
+from py42 import exceptions
+
+from code42cli.bulk import run_bulk_process
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.errors import Code42CLIError
+from code42cli.file_readers import read_csv_arg
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.output_formats import OutputFormatter
+
+
+@click.group(cls=OrderedGroup)
+@sdk_options(hidden=True)
+def devices(state):
+    """For managing computers within Code42."""
+    pass
+
+
+device_id_option = click.option(
```

Comment:
This needs to call out that is a legacy ID..

It is unfortunate though that we have to use legacy IDs.

I am wondering what thoughts are for making these be the normal device GUIDs and just getting the legacy ID behind the scenes.

Reply:
The underlying method in py42 uses legacy ID, as does the API it calls. I agree that it would be best if we could use GUID, but is it best to have that translation live here? Or should it be done in py42?

Reply:
It would be a breaking change in py42, so maybe it is a discussion for py42 2.0, unless the methods support both. I think my favorite option is do this in the CLI for now.

Reply:
I agree that the bulk command should continue to use the legacy Id. We make sure to should call that out.

Reply:
Updated `devices show` and `devices deactivate` to use GUID. Clarified the docstring in `devices bulk deactivate` that it does not use GUID, and instead uses legacy deviceId.